wordpress-mobile/WordPress-iOS
# Add Delete Site Analytics

Fixes #4950

To test:
1) Install current WordPressCom-Analytics-iOS 0.1.5 candidate and ensure compilation shows all new cases are handled.
2) Verify new cases are defined/handled consistently with Android usage.

Needs review: @astralbodies
